.travis.yml

```diff
@@ -46,8 +46,8 @@ jobs:
           install:
               - pip3 install --upgrade pip
               - pip3 install --progress-bar off poetry
-              - python3 -m poetry --no-interaction install
-              - python3 -m poetry --no-interaction build
+              - python3 -m poetry install --no-interaction
```

Comment:
Why? What is the purpose of these changes?

Reply:
Same as https://github.com/datadvance/DjangoChannelsGraphqlWs/pull/37/commits/288f9e75e34a6d3f38d9eda10ee8b39b80125741#r403520450